src/js/components/cycleCount/CycleCount.jsx

```diff
@@ -40,6 +41,25 @@ const CycleCount = () => {
     isLoading,
   } = useCycleCountFilters();
 
+  const [totalCount, setTotalCount] = useState(0);
+
+  // Moved this here to prevent resetting number of rows per page when switching tabs.
+  const {
+    paginationProps,
+    offset,
+    pageSize,
+  } = useTablePagination({
+    defaultPageSize: 5,
+    totalCount,
+    filterParams,
+  });
+  const tablePaginationProps = {
+    paginationProps,
+    offset,
+    pageSize,
+    setTotalCount,
+  };
```

Comment:
let's move those lines into a separate hook, something like `useCycleCountPagination`, to avoid having too much logic (such as useStates) in the component.